web/src/components/assets/AssetCreationForm.tsx

```diff
@@ -411,26 +393,29 @@ export default function AssetForm(props: AssetFormProps) {
             className={errors.status ? styles.error : ""}
           >
             <option value="">Select status</option>
-            <option value="Active">Active</option>
-            <option value="Awaiting">Awaiting</option>
-            <option value="Destroyed">Destroyed</option>
+            <option value="active">Active</option>
+            <option value="awaiting">Awaiting</option>
+            <option value="destroyed">Destroyed</option>
```

Comment:
might be beneficial to expand slightly on what awaiting means (what does it mean?)


also is it likely they will create an asset that's destroyed? This could just be an option for the update form

Reply:
Any suggestions on the wording? Perhaps @finleybacon has some ideas?

Reply:
maybe something like
`Active: asset is in environment`
`Awaiting: asset awaiting creation in environment`
`Destroyed: asset has been destroyed in environment`

?

still not really sure of the merit of being able to select destroyed on creation tho